doc/api/test.md

```diff
@@ -1575,10 +1789,12 @@ Mocking timers is a technique commonly used in software testing to simulate and
 control the behavior of timers, such as `setInterval` and `setTimeout`,
 without actually waiting for the specified time intervals.
 
+MockTimers is also able to mock the `Date` object.
+
 The [`MockTracker`][] provides a top-level `timers` export
 which is a `MockTimers` instance.
 
-### `timers.enable([timers])`
+### `timers.enable([EnableOptions])`
 
 <!-- YAML
```

Comment:
We need to add the `changes` documentation to the `<!-- YAML`>  since you are changing the API definition. Example:

```
changes:
  - version:
    - REPLACEME
    pr-url: https://github.com/nodejs/node/pull/48638/
    description: Add a short description here
```

Reply:
Also, feel free to ignore my request since you have been fighting with the CI for so long. Free free to send a follow-up PR.

Reply:
Will do the change here and commit on this PR

Reply:
Done in the last commit :)